src/github.com/travis-ci/jupiter-brain/vsphere.go

```diff
@@ -301,10 +302,27 @@ func (i *vSphereInstanceManager) createClient(ctx context.Context) (*govmomi.Cli
 	client.Client.RoundTripper = &soapBreakerRoundTripper{
 		rt: client.Client.RoundTripper,
 		cb: gobreaker.NewCircuitBreaker(gobreaker.Settings{
-			Name: "vSphere govmomi",
+			Name:     "vSphere govmomi",
+			Interval: 10 * time.Minute,
 			ReadyToTrip: func(counts gobreaker.Counts) bool {
 				failureRatio := float64(counts.TotalFailures) / float64(counts.Requests)
-				return counts.Requests >= 3 && failureRatio >= 0.6
+				return counts.Requests >= 5 && failureRatio >= 0.6
```

Comment:
is there a scientific reason for ~60% failures, or is this just an assumption? this might be worthwhile to mention.